core/src/main/java/org/apache/iceberg/ManifestEntriesTable.java

```diff
@@ -104,8 +104,8 @@ protected TableScan newRefinedScan(TableOperations ops, Table table, Schema sche
     }
 
     @Override
-    protected long targetSplitSize(TableOperations ops) {
-      return ops.current().propertyAsLong(
+    public long targetSplitSize() {
```

Comment:
I think it would be better to separate this refactor into its own PR.

Reply:
Good idea. Will do.

Reply:
I opened #2299